pkg/defkinds/nodejs/definition_test.go

```diff
@@ -175,6 +201,45 @@ func initParseRawDefinitionWithWebserverTC() newDefinitionTC {
 	}
 }
 
+func initParseRawDefinitionWithCustomHealthcheckTC() newDefinitionTC {
+	devStageDevMode := true
+	prodStageDevMode := false
+
+	baseStage := emptyStage()
+	baseStage.Healthcheck = &builddef.HealthcheckConfig{
+		HealthcheckHTTP: &builddef.HealthcheckHTTP{
+			Path:     "/some-custom-path",
+			Expected: "some-output",
+		},
+		Type:     builddef.HealthcheckTypeHTTP,
+		Interval: 20 * time.Second,
+		Timeout:  5 * time.Second,
+		Retries:  6,
+	}
+
+	return newDefinitionTC{
+		file: "testdata/def/with-custom-http-healthcheck.yml",
+		expected: nodejs.Definition{
```

Comment:
shouldn't the HC config also be asserted here ?

Reply:
It's already the case but the BaseStage is declared in a var. I'm going to change that to write these tests like other tests in definition_test.go files :wink: